node/src/actors/chain_manager/mining.rs

```diff
@@ -649,9 +705,19 @@ fn build_block(
     }
 
     for ta_tx in tally_transactions {
-        if let Some(dr_output) = dr_pool.get_dr_output(&ta_tx.dr_pointer) {
+        if let Some(dr_state) = dr_pool.data_request_state(&ta_tx.dr_pointer) {
             tally_txns.push(ta_tx.clone());
-            transaction_fees += dr_output.tally_fee;
+            let commits_count = dr_state.info.commits.len();
+            let reveals_count = dr_state.info.reveals.len();
+            let honests_count = commits_count - ta_tx.slashed_witnesses.len();
+            // Remainder collateral goes to the miner
```

Comment:
Did we miss mentioning this in WIP-0002?

Reply:
Yes, we should include in the WIP-0002. I didnt saw but I rememeber that we discussed at the office

Reply:
I'll be updating `WIP-0002`. Thanks for the clarification @lrubiorod .